lib/pages/people/view/person_banner.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:acs_upb_mobile/pages/people/view/person_avatar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_feather_icons/flutter_feather_icons.dart';
+
+import 'package:acs_upb_mobile/widgets/icon_text.dart';
+
+class PersonBanner extends StatelessWidget {
+  const PersonBanner({
+    @required this.name,
+    @required this.photoURL,
+    @required this.position,
+    @required this.office,
```

Comment:
Why not just pass a Person directly?

Reply:
Because, imo, the banner should only know how to handle _this specific information_, not the entire data stored in a Person.
Also, widgets like `ContactInfo` might be used in other circumstances.

Reply:
But, in this case, it may be better to have the `PersonBanner` class private and nested in the `PersonView` as its only usability is here

Reply:
Well, for other use cases, you'd still show a *person* most likely, not some other random data. But in general, whenever you think of a sentence like, "this might be used...", you should also remember YAGNI.

I agree with your second point though, this could be either a private widget in PersonView or simply a method in PersonView that returns a widget.

Reply:
https://github.com/student-hub/acs-upb-mobile/pull/252/commits/396dff90559907416f9ea31238fb87d37c747e59 moved the `PersonBanner` inside `person_view.dart`